skll/learner/__init__.py

```diff
@@ -734,23 +764,24 @@ def _check_max_feature_value(self, feat_array):
                 "perform poorly."
             )
 
-    def _create_label_dict(self, examples):
+    def _create_label_dict(self, examples: FeatureSet) -> None:
         """
         Create a dictionary of labels for classification problems.
 
         Parameters
         ----------
-        examples : skll.FeatureSet
+        examples : skll.data.FeatureSet
             The examples to use for training.
         """
         # we don't need to do this if we have already done it
         # or for regression models, so simply return.
-        if self.label_dict is not None or self.model_type._estimator_type == "regressor":
+        if len(self.label_dict) > 0 or self.model_type._estimator_type == "regressor":
```

Comment:
should check that self.label_dict is None first, or is it impossible?

Reply:
It won't ever be `None` since the default value now is an empty dictionary. I changed it in this PR.